sa.sy.bluerecorder.yaml

```diff
@@ -0,0 +1,79 @@
+app-id: sa.sy.bluerecorder
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+cleanup:
+  - /share/man
+  - /share/doc
+  - /share/gtk-doc
+command: /app/opt/blue-recorder
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=x11
```

Comment:
if you have wayland support, this should be `--socket=fallback-x11`

Reply:
done